requirements_gui.txt

```diff
@@ -1,7 +1,7 @@
-# fixes a memory leak. TODO: update to 4.7.3 when released
-git+git://github.com/enthought/enable@2c0a312f0b56a#egg=enable
+# TODO: move back to enthought (upstream) versions once they add python 3.7 support
+git+git://github.com/swift-nav/enable@swift-2019.04#egg=enable
```

Comment:
https would be better for public repos to not to force ssh key setup?

Reply:
We definitely don't want to require ssh key setup but I don't think this usage in requirements.txt needs it...

Reply:
I added a tag `4.7.0.dev64+swift0` so we can pin to a specific commit without a git hash and not use the branch name.

Reply:
```suggestion
git+git://github.com/swift-nav/enable@4.7.0.dev64+swift0#egg=enable
```